### DISCUSSION TITLE
Getting the stock quantity of a specific product

### DISCUSSION THREAD

**Discussion by ayoubAnbara** [amzn/selling-partner-api-models#343] [Q&A]
I want to Get the stock quantity of a specific product by SKU or ASIN without using Reports API.

**Comment by leonardonakagawa** [comment]
Hello @ayoubAnbara,

Thank you for reaching out.

You can get the inventory quantity by using the getListingsItem API. When using it you can leverage on passing the `SKU` information and  with the `includedData` parameter, you can include the value `fulfillmentAvailability`. This parameter allows you to retrieve the fulfillment availability details for the listing item.

You can find details regarding this in our [Listings Items API v2021-08-01 Use Case Guide](https://developer-docs.amazon.com/sp-api/docs/listings-items-api-v2021-08-01-use-case-guide#step-1-submit-listings-item-get-request) and [IncludedData](https://developer-docs.amazon.com/sp-api/docs/listings-items-api-v2021-08-01-reference#includeddata) information.
If the information available in the documentation link provided above does not fully resolve your inquiry, please [open a support case](https://developer.amazonservices.com/support) with us.

Thanks,
Leonardo Nakagawa
Solutions Architect, Selling Partner API Developer Support
